src/app/services/updateRapid.js

```diff
@@ -1,13 +1,14 @@
 import http from "@/app/http"
 
 export async function updateRapid(data) {
-  const point = `${data.geom.coordinates[0]} ${data.geom.coordinates[1]}`;
-
   const payload = {
-    ...data,
-    rloc: point,
+    ...data
```

Comment:
If we're not changing `data` we could just pass it directly without copying entries.

Reply:
oh I see like I could just put rloc into the query directly, and use data as-is for the rest. copy that